lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/passport/issuecredential/IssueCredentialHandler.java

```diff
@@ -251,14 +251,9 @@ private SignedJWT generateAndSignVerifiableCredentialJwt(
                         .issuer(configurationService.getSsmParameter(VERIFIABLE_CREDENTIAL_ISSUER))
                         .audience(configurationService.getClientIssuer(passportCheck.getClientId()))
                         .notBeforeTime(new Date(now.toEpochMilli()))
-                        .expirationTime(
-                                new Date(
-                                        now.plusSeconds(
-                                                        Long.parseLong(
-                                                                configurationService
-                                                                        .getSsmParameter(
-                                                                                MAX_JWT_TTL)))
-                                                .toEpochMilli()))
+                        .claim(
```

Comment:
Do we not want to have this as .expirationTime still?

Reply:
Expiration time has to be a date which were not using. It just does the EXPIRATION_TIME claim behind the scenes